ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/forms/CollectionSelection.tsx

```diff
@@ -167,24 +177,32 @@ function CollectionSelection({
                                 </SelectOption>
                             ))}
                         </Select>
-                    </FormLabelGroup>
-                </FlexItem>
-                {allowCreate && (
-                    <FlexItem>
+                    </FlexItem>
+                    <FlexItem spacer={{ default: 'spacerMd' }}>
                         <Button
-                            className="pf-u-mb-md"
-                            variant={ButtonVariant.secondary}
-                            onClick={onToggleCollectionModal}
+                            variant={ButtonVariant.tertiary}
+                            onClick={onOpenViewCollectionModal}
+                            isDisabled={selectedScopeId === ''}
                         >
-                            Create collection
+                            View
                         </Button>
```

Comment:
Added a `View` button to view the selected report scope (collection). I also changed the layout to make it look like this 

<img width="449" alt="Screenshot 2023-06-29 at 3 13 15 PM" src="https://github.com/stackrox/stackrox/assets/4805485/39fe1fe6-20ec-4fad-99f0-2a56c98503ec">